src/main/webapp/city.js

```diff
@@ -63,12 +63,12 @@ function displayCityResults(city, country) {
 
   // add language spoken to "dos" list
   const languageElement = document.createElement('li');
-  languageElement.appendChild('Learn a few words in ' + country.languages);
+  languageElement.innerText('Learn a few words in ' + country.languages);
```

Comment:
Needs to be `.innerText = ` like here:
https://github.com/gferioli0418/travelbud/blob/master/src/main/webapp/city.js#L40